### PR TITLE
Initialize repository

### PR DESCRIPTION
**Pull request by prog893**
eden 3/3 - API flavor with Lambda+ALB

### CODE REVIEW

File: .terraform/modules/34fda75a37d4cc8fcebdb106c425c61d/baikonur-oss-terraform-aws-iam-nofile-11a0b7b/.gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### macOS
```
Comment (toVersus) [inline .terraform/modules/34fda75a37d4cc8fcebdb106c425c61d/baikonur-oss-terraform-aws-iam-nofile-11a0b7b/.gitignore]: Hmm, it seems that files under .terraform are included in this PR. Please remove them by running `git rm --cached`?

Comment (prog893) [reply .terraform/modules/34fda75a37d4cc8fcebdb106c425c61d/baikonur-oss-terraform-aws-iam-nofile-11a0b7b/.gitignore]: done

File: README.md [hunk 2]
```diff
@@ -1,17 +1,127 @@
...
+2019-04-10T23:14:46.514Z INFO     [main.py:delete_cname_record:600] CNAME record for api-add-nothing.dev.example.com does not exist, skipping deletion 
+2019-04-10T23:14:46.872Z INFO     [main.py:delete_env:662] ECS Service dev-dynamic-add-nothing not found, skipping deletion 
+2019-04-10T23:14:46.923Z INFO     [main.py:delete_env:691] Target group dev-dynamic-add-nothing not found, skipping deletion of listener rule and target group 
+2019-04-10T23:14:46.991Z INFO     [main.py:delete_env:697] Deleted all task definitions for family: dev-dynamic-add-nothing, 0 tasks deleted total 
+2019-04-10T23:14:46.991Z INFO     [main.py:delete_env:700] Successfully finished deleting environment dev-dynamic-add-nothing 
+```
+
 ### Version pinning
 #### Terraform Module Registry
 Use `version` parameter to pin to a specific version, or to specify a version constraint when pulling from [Terraform Module Registry](https://registry.terraform.io) (`source = baikonur-oss/%module_name%/aws`).
```
Comment (toVersus) [inline README.md]:
```suggestion
Use `version` parameter to pin to a specific version, or to specify a version constraint when pulling from [Terraform Module Registry](https://registry.terraform.io) (`source = baikonur-oss/terraform-aws-lambda-eden-api/aws`).
```

**Review by toVersus — APPROVED** [review]
LGTM

**Pull request merged**